server/datastore/mysql/teams.go

```diff
@@ -52,7 +57,7 @@ func (ds *Datastore) Team(ctx context.Context, tid uint) (*fleet.Team, error) {
 
 func teamDB(ctx context.Context, q sqlx.QueryerContext, tid uint) (*fleet.Team, error) {
 	stmt := `
-		SELECT * FROM teams
+		SELECT ` + teamColumns + ` FROM teams
```

Comment:
Is this for optimization? or other need?

Reply:
I added a new column used only for indexing: `name_bin`, so we do not need to get it from DB. In fact, if we get it, we will not be able to unmarshal it. The same thing is done for the `checksum` column in the software table. So, this change is needed to make this SELECT statement work.